dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/SdkFeature.kt

```diff
@@ -319,6 +359,8 @@ internal class SdkFeature(
     // endregion
 
     companion object {
+        internal const val CONTEXT_UPDATE_LISTENER_ALREADY_EXISTS =
+            "Feature \"%s\" already has this listener registered, overwriting it."
```

Comment:
Still there is no overwrite, it is already there. Maybe we can drop the part about `overwriting`?